recipes/e3sm-simple/build.bash

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+#conda build -c conda-forge -c defaults -c e3sm -c cdat/label/v81 meta.yaml
```

Comment:
```suggestion
conda build -c conda-forge -c defaults meta.yaml
```

Reply:
This probably shouldn't be commented out by default (I know you're following my lead), but I shouldn't have committed it commented out.

Reply:
Done! See dcd9ac9c966474cc3132dccabbc8032fecd2ec87